File: cplp_content/prueba.pas
Program P                           // Sintáctico: Falta ;
    var 5: integer;                 // Sintáctico: No se puede tener un número como nombre de una variable
    var a:char;
Begin
    for i:=5 to 10 do begin         // Semántico: La variable "i" no está declarada. Detectado en Compilación.
        write(a);                   // Semántico: La variable "a" no está inicializada. Detectado en Compilación.
        a=a+1;                      {
                                    Sintáctico: Es a:=a+1;
                                    Semántico: La variable "a" es de tipo "char", sumarle
                                    1 no es una operación válida. Detectado en Compilación.
                                    } 
    end;
End.

File: cplp_content/prueba.java
public String tabla(int numero, arrayList<Boolean> listado) {       // Sintáctico: Es ArrayList.
    String result = null;
    for(i = 1; i < 11; i--) {                                       /* 
                                                                    Lógico: Loop infinito.
                                                                    Semántico: La variable "i" no está
                                                                    declarada. Detectado en Compilación.
                                                                    */ 
        result += numero + "x" + i + "=" + (i*numero) + "\n";       /*
                                                                    Semántico: Concatenación de null en la
                                                                    cadena. Detectado en Compilación.
                                                                    */ 
        listado.get(listado.size()-1)=(BOOLEAN) numero>i;           /*
                                                                    Sintáctico: El lado izquierdo de la
                                                                    asignación debe ser una variable.
                                                                    Sintáctico: "BOOLEAN" no existe, debería
                                                                    ser Boolean o boolean.
                                                                    */
    }
    return true;                                                    /*
                                                                    Semántico: Error de tipo en el return. 
                                                                    Detectado en Compilación.
                                                                    */
}